src/ray/rpc/node_manager/node_manager_client.h

```diff
@@ -115,6 +115,9 @@ class NodeManagerWorkerClient
   /// Get gcs server address.
   VOID_RPC_CLIENT_METHOD(NodeManagerService, GetGcsServerAddress, grpc_client_, )
 
+  /// A worker is blocked.
+  VOID_RPC_CLIENT_METHOD(NodeManagerService, DirectCallTaskBlocked, grpc_client_, )
```

Comment:
Could you give me some explanation about why we need this one? I can't link it with the queue here easily.